snippets/go/connecting_lsp.go

```diff
@@ -16,6 +16,14 @@ func GetLspInfo() {
 	// ANCHOR_END: get-lsp-info
 }
 
+func ListLsps() {
+	// ANCHOR: list-lsps
+	if err := sdk.ListLsps(); err != nil {
+		log.Printf("%#v", err)
+	}
```

Comment:
```suggestion
	if lspList = err := sdk.ListLsps(); err != nil {
		log.Printf("%#v", err)
	}
```

Reply:
Sorry I missed this, I think this is more functional

```suggestion
	if lspList, err := sdk.ListLsps(); err == nil {
		log.Printf("%#v", lspList)
	}
```
or if the func has to be functional
```
func ListLsps() ([]breez_sdk.LspInformation, error) {
    // ANCHOR: list-lsps
    lspList, err := sdk.ListLsps()
    if err != nil {
	return []breez_sdk.LspInformation{}, err
    }
    // ANCHOR_END: list-lsps
    return lspList, nil
}
```

Reply:
Done, thanks!